app/src/main/res/menu/home_menu.xml

```diff
@@ -50,6 +50,10 @@
       android:title="@string/menu_jump"
       app:showAsAction="never" />
   <item
+      android:id="@+id/translations"
+      android:title="Translations"
```

Comment:
Hmm... there should not be. Android Studio shows them as string literals, but actually they are not. 

Overall, you will have to redo this patch anyway. Don't put the menu here. "More translations" option should go to translations spinner in translation screen/mode, the one you see at the top when you have more than one translation are installed.